cmd/bosun/sched/check.go

```diff
@@ -323,10 +329,35 @@ func (s *Schedule) CollectStates() {
 		ackStatusCounts[alert.Name][true] = 0
 		activeStatusCounts[alert.Name][true] = 0
 	}
+	for notificationName, _ := range s.Conf.Notifications {
+		unAckOldestByNotification[notificationName] = time.Unix(1<<63-62135596801, 999999999)
```

Comment:
Mac value of time.Time, so I can loop and get the min. http://stackoverflow.com/a/32620397/107156